tests/test_tracing.py

```diff
@@ -96,6 +96,58 @@ def _wrap(*args, **kwargs):
     return _wrapper
 
 
+_test_extract_dd_trace_context = (
+        ("api-gateway", Context(trace_id=12345, span_id=67890, sampling_priority=2)),
```

Comment:
Question, is there any difference if we just use the same Context object for all of them? I.e. data is the same, as opposed to changing the trace_id, span_id and sampling_priority all the time? 

I see some of them have different sampling priority, but I don't understand why they have different trace_id/span_id, is it important in those cases?

Reply:
I suspect the reason why the trace/span ids are different is because these values are pulled from the event json itself from the text fixture file. Bringing these into alignment would mean changing those fixtures and could have impacts on other tests as well.

Reply:
Thanks – makes sense, appreciate it